js_modules/dagit/packages/ui/src/components/Dialog.tsx

```diff
@@ -61,6 +61,26 @@ export const DialogBody: React.FC = (props) => {
   );
 };
 
+// The subheadings used in the backfill modals
+export const DialogBodySection = ({
+  title,
+  children,
+}: {
+  title: string | React.ReactNode;
+  children: React.ReactNode;
+}) => (
+  <Box flex={{direction: 'column', gap: 8}}>
+    <strong style={{display: 'block'}}>{title}</strong>
+    <Box
+      flex={{direction: 'column', gap: 8}}
```

Comment:
Are you certain you always want the children to be flex-column items?

Reply:
Hmm will move this back to being in the materialize modal so we can keep it specific to that UI